edd-taxjar.php

```diff
@@ -1,12 +1,12 @@
 <?php
-/*
-Plugin Name: Easy Digital Downloads - TaxJar
-Plugin URI: https://easydigitaldownloads.com/extensions/taxjar
-Description: Calculate sales tax through automatically TaxJar.com
-Version: 1.0.1
-Author: Easy Digital Downloads
-Author URI: https://easydigitaldownloads.com
-*/
+/**
+ * Plugin Name: Easy Digital Downloads - TaxJar
+ * Plugin URI: https://wordpress.org/plugins/edd-taxjar/
+ * Description: Calculate sales tax through automatically TaxJar.com
```

Comment:
```suggestion
 * Description: Calculate sales tax automatically through TaxJar.com
```

Sorry, just realized that was wonky.